lib/shopify/customer.ex

```diff
@@ -64,7 +104,29 @@ defmodule Shopify.Customer do
   end
 
   @doc """
-  Retrieve all orders belonging to a customers.
+  Retrieve a single customer metafield
+  """
+  @spec get_customer_metafield(binary, binary) :: Shopify.Operation.t()
+  def get_customer_metafield(customer_id, metafield_id) do
```

Comment:
Since this is on the `Customer` module adding `customer` to the function name is redundant. This can just be `get_metafield/2`.